crt_portal/cts_forms/templates/forms/grouped_questions.html

```diff
@@ -0,0 +1,46 @@
+{% for question_group in wizard.form.question_groups %}
+  <fieldset class="usa-fieldset margin-bottom-5">
+    <!-- TODO: REMOVE using an h2 here over a legend allows us to benefit from the auto styling of 
```

Comment:
Hmmm ... I don't see an `<h2>` or a `<legend`> in this file, so should this code comment move somewhere else? Does `TODO: REMOVE` mean that you want to remove this comment before merging the PR?

Reply:
Yes, the todo is a note to myself to remove this before the WIP status is removed. There _was_ an `h2`, (now an `h3`), but the comment relates to `h` tags in general 😄